.github/workflows/hub_push.yml

```diff
@@ -0,0 +1,82 @@
+name: Push All Agents to Hub
+
```

Comment:
lets add a TODO note here to indicate that the pipeline does not work until CLI-driven login is implemented on hub/dir